src/tools/installation/yaml/manager.yaml

```diff
@@ -5,6 +5,757 @@ metadata:
     control-plane: cnsi-controller
   name: cnsi-system
 ---
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
+metadata:
+  annotations:
+    controller-gen.kubebuilder.io/version: v0.4.1
+  creationTimestamp: null
+  name: inspectionpolicies.goharbor.goharbor.io
+spec:
+  group: goharbor.goharbor.io
+  names:
+    kind: InspectionPolicy
+    listKind: InspectionPolicyList
```

Comment:
The most important reason for putting the crd here is:

This file is autogenerated, so we an eliminate manually copy and paste.